data-pipeline/assessment-aggregator/src/main/java/org/ekstep/ep/samza/service/AssessmentAggregatorService.java

```diff
@@ -0,0 +1,82 @@
+package org.ekstep.ep.samza.service;
+
+import com.datastax.driver.core.Row;
+import com.datastax.driver.core.UDTValue;
+import com.google.gson.Gson;
+import com.google.gson.JsonSyntaxException;
+import org.ekstep.ep.samza.core.Logger;
+import org.ekstep.ep.samza.domain.Aggregate;
+import org.ekstep.ep.samza.domain.BatchEvent;
+import org.ekstep.ep.samza.domain.QuestionData;
+import org.ekstep.ep.samza.task.AssessmentAggregatorSink;
+import org.ekstep.ep.samza.task.AssessmentAggregatorSource;
+import org.ekstep.ep.samza.util.DBUtil;
+import org.joda.time.DateTime;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class AssessmentAggregatorService {
+
+    private static Logger LOGGER = new Logger(AssessmentAggregatorService.class);
+    private DBUtil dbUtil;
+
+    public AssessmentAggregatorService(DBUtil dbUtil) {
+        this.dbUtil = dbUtil;
+    }
+
+    public void process(AssessmentAggregatorSource source, AssessmentAggregatorSink sink) {
```

Comment:
please add metrics to count:
1. no. of batchs
2. count of individual events in a batch